app/js/questionNav.js

```diff
@@ -1,55 +1,89 @@
 var current = 1
 
 /**
-*adds active to the first question
-* function in called when questions are inserted into html
+ * adds active to the first question
```

Comment:
doesnt explain the new functionality of what `trackActiveQuestion()` does

Reply:
corrected in 78abf07